oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -182,6 +193,7 @@ int pldm::responder::oem_ibm_platform::Handler::
             }
             else if (stateSetId == PLDM_OEM_IBM_SLOT_ENABLE_EFFECTER_STATE)
             {
+                std::cerr << "Effecter id : " << effecterId << std::endl;
```

Comment:
please remove this trace , as this is not related

Reply:
done